packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -87,23 +90,51 @@ export enum BrandingStyle {
   Logo_Slogan = 'Logo_Slogan',
 }
 
-export const brandingGuard = z.object({
-  primaryColor: z.string(),
-  backgroundColor: z.string(),
-  darkMode: z.boolean(),
-  darkPrimaryColor: z.string(),
-  darkBackgroundColor: z.string(),
-  style: z.nativeEnum(BrandingStyle),
-  logoUrl: z.string().optional(),
-  slogan: z.string().optional(),
-});
+export const brandingGuard = z
+  .object({
+    primaryColor: z
+      .string()
+      .nonempty()
+      .refine((color) => isValidHexColor(color), invalidHexColorMessage),
+    backgroundColor: z
+      .string()
+      .nonempty()
+      .refine((color) => isValidHexColor(color), invalidHexColorMessage),
```

Comment:
Lets discuss whether is a good practice to assert the custom refine validations here.